bin/node/runtime/src/lib.rs

```diff
@@ -1256,6 +1265,7 @@ construct_runtime!(
 		Indices: pallet_indices,
 		Balances: pallet_balances,
 		TransactionPayment: pallet_transaction_payment,
```

Comment:
Just as a note: this stays, we keep the pallet, just the SignedExtension is replaced.